sdk/sdkintegrations/initconn.go

```diff
@@ -56,11 +56,9 @@ func (c ConnectionInit) AbortWithStatus(status int, err string) {
 		u := "vscode://autokitteh.autokitteh?cid=%s&status=%d&error=%s"
 		u = fmt.Sprintf(u, c.ConnectionID, status, url.QueryEscape(err))
 		http.Redirect(c.Writer, c.Request, u, http.StatusFound)
-	case "web":
-		http.Error(c.Writer, err, status)
```

Comment:
@daabr we need those lines, otherwise, I don't get the error correctly on our SaaS. It sends back the whole HTML of the error page redirected to.

Reply:
Ack. Removing these 2 lines simply means that `"web"` is handled by the `default` block below.

Specifically, it will return an HTML page of the error, but as requested by @itay it will also return the right HTTP status code - see `web/connect/result/handler.go` line 77.

Reply:
Yeah, but this way I don't get the error, I can't take it from the HTML file.

Reply:
Summary of offline discussion: bringing back these 2 lines. They are useful whenever the user is creating a non-OAuth connection via SaaS. The difference is that non-OAuth connections don't open a new browser tab, so they do expect a machine-readable response.

However, this means that AK will need to distinguish between OAuth and non-OAuth in "web" origins. OAuth origins need to be treated like non-"web", i.e. result in an user-friendly HTML page, because we do open them in separate browser tabs.

Furthermore, in a future PR, I will add a SaaS base URL config value, which will allow us to redirect non-OAuth "web" errors back to the SaaS (because non-OAuth connections don't open a new browser tab).

@RonenMars - note that this means you'll have to parse a status code and error message in the URL instead of the body, but I assume that's still better than parsing an HTML page. This is because if AK redirects the response back to the SaaS frontend, it can't set an HTTP status or an HTTP response body.

Please acknowledge.

Reply:
> Ack. Removing these 2 lines simply means that `"web"` is handled by the `default` block below.
> 
> 
> 
> Specifically, it will return an HTML page of the error, but as requested by @itay it will also return the right HTTP status code - see `web/connect/result/handler.go` line 77.

FYI I think you meant a different Itay 😀

Reply:
Oops! Sorry for the typo, indeed I meant @itayd. But this is a friendly neighborhood, feel free to hang around. 😄

Reply:
FYI for @itayd - as a final step (in a separate PR), @RonenMars will change the FE's OAuth connection flow to redirect instead of open a new tab, and I will change AK to redirect back to the SaaS instead of AK (which currently results in responding with an HTML page).